xml/System.Collections/Hashtable.xml

```diff
@@ -98,6 +98,7 @@
  [!code-cpp[System.Collections.Hashtable_ClassExample#00](~/samples/snippets/cpp/VS_Snippets_CLR_System/system.Collections.Hashtable_ClassExample/cpp/hashtable_example.cpp#00)]
  [!code-csharp[System.Collections.Hashtable_ClassExample#00](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.Collections.Hashtable_ClassExample/cs/hashtable_example.cs#00)]
  [!code-vb[System.Collections.Hashtable_ClassExample#00](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.Collections.Hashtable_ClassExample/vb/hashtable_example.vb#00)]  
+ [!code-powershell[System.Collections.Hashtable_ClassExample#00](~/samples/snippets/powershell/VS_Snippets_CLR_System/system.Collections.Hashtable_ClassExample/ps/hashtable_example.ps1#00]
```

Comment:
@doctordns, the code example isn't appearing in the build (the file reference can't be resolved) because of a case mismatch;  you have to change /ps/ in the path here to uppercase (/PS/) for the file to be found.